### ISSUE TITLE
validationResult called in another package

### ISSUE THREAD

**Issue by grayhound** [express-validator/express-validator#736]
For some reason, if validationResult is called from another npm package, it returns empty errors list.

Here's how to reproduce.

- Routes:

``` 
  const usersController = new UsersController();

  router
    .route('/users/signup')
    .post(
      [
        body('email').not().isEmpty(),
        body('email').isEmail(),
      ]
      usersController.validate.bind(usersController)
    );
```
- UsersController

```
export default class UsersController extends BasicController {
  ...
}
```
- BasicController - this one is placed in **OTHER** npm package.
```
const { validationResult } = require('express-validator');

export default class BasicController {
  validate(req, res, next) {
    const errors = validationResult(req);
    if (!errors.isEmpty()) {
      console.log('Validation errors, better return HTTP 422');
    }
    next();
  }
}
```

Errors are empty no matter what.

As soon as move `validate` method from `BasicController` to `UsersController` (so, validationResult is in the same package with middlewares/routes) - everything works correctly.

This can be reproduced with `6.1.1`, but everything works just fine with `5.3.1`.

NPM package with BasicController is built with `@babel` using `@babel/preset-env` preset.

Am I missing something? Any reason behind it?

**Comment by gustavohenke** [comment]
I reckon you don't have `express-validator` deduped.
Both versions should be in sync, as these is touching internals that have changed.

**Comment by npalmius** [comment]
Is this is a duplicate of #813? So resolved in v6.3.1? I was having the same problem (although only when using `npm link`ed package).

**Comment by gustavohenke** [comment]
It must have been 👍
Upgrading will definitely help @grayhound
